### ISSUE TITLE
[v0.9] creating new shellsensor throws error

### ISSUE THREAD

**Issue by leader21** [pimatic/pimatic#915]
![](http://www.youscreen.de/osoumsamh00.jpg)

![](http://www.youscreen.de/rfkehwogq69.jpg)

**Comment by leader21** [comment]
i guess this has something to do with this issue
https://github.com/pimatic/pimatic/issues/867

i created a simple date device and inserted it manually to the config. worked flawlessly
```
    {
      "id": "date",
      "name": "Heute",
      "class": "ShellSensor",
      "attributeName": "today",
      "attributeType": "string",
      "command": "date +'%H:%M Uhr, %d.%m.%Y'",
      "interval": 60000
    },
```
then i tried to edit this device - worked without problems. 

here comes the recreatable issue!
go to a variables device and click on the variable for to get into the settings. save the settings and save the variables device. then create a new device (shellsensor) and save it. you don't even have to insert a command. you will get the above error or like the error below 

![](http://www.youscreen.de/mzibwpniw21.jpg)

**Comment by sweetpi** [comment]
Should be fixed in pimatic-shell-execute 0.9.4.
